Uzduotis25/Program.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+
+namespace Uzduotis25
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            // Liepti vartotojui įvesti 5 pažymius VIENOJE eilutėje atskiriant tarpu.
+            // Rasti  didžiausią pažymį.
+
+            Console.Write("Enter 5 grades separated by a space: ");
+            string input = Console.ReadLine();
+
+            string[] gradeStrings = input.Split(' ');
+
+            if (gradeStrings.Length != 5)
+            {
+                Console.WriteLine("Please enter exactly 5 grades separated by a space.");
+            }
+            else
+            {
+                int[] grades = new int[5];
```

Comment:
Darote išvedimą vartotojui, kad būtinai turi įvesti 5 reikšmes, nes kuriate masyvą 5 reikšmių dydžio. Bet su komanda `Split` jau gaunate masyvą ir aiškus jos dydis taigi čia būtų galima panaudoti to masyvo dydį.